.github/workflows/build-and-test.yml

```diff
@@ -193,6 +193,12 @@ jobs:
           # natcap.invest from source and that it imports.
           python -c "from natcap.invest import *"
 
+      - name: Set up environment
+        run: pip install twine

```

Comment:
Would it be worth including `twine` under the `requirements` list up above, when we're using the actions `setup_env` step?  I don't think it'd be strictly necessary, but I think it might avoid the need to re-resolve the dependencies, which could potentially save some small amount of time.  I think the bigger benefit might be just having all of the python dependencies in once place.

So I'm thinking something like this up above:
```
      - uses: ./.github/actions/setup_env
        with:
          python-version: ${{ matrix.python-version }}
          requirements-files: requirements.txt
          requirements: ${{ env.CONDA_DEFAULT_DEPENDENCIES }} twine
```

Reply:
Great idea, thanks!